project/deps/package.mill.scala

```diff
@@ -75,9 +77,10 @@ object Scala {
   def maxAmmoniteScala213Version  = scala213
   def maxAmmoniteScala3Version    = "3.6.3"
   def maxAmmoniteScala3LtsVersion = "3.3.5"
-  lazy val listMaxAmmoniteScalaVersion =
+  lazy val listMaxAmmoniteScalaVersion: Seq[String] =
     Seq(maxAmmoniteScala212Version, maxAmmoniteScala213Version, maxAmmoniteScala3Version)
-  lazy val listAllAmmonite = {
+  @unused
+  lazy val listAllAmmonite: Seq[String] = {
```

Comment:
is it an error for unsued or is it indeed never used?

Reply:
it's so that the IDE doesn't complain that it's unused.
IntelliJ doesn't understand how Mill commands work.

Reply:
it's just IntelliJ. Turning it off requires putting in a special comment or turning the functionality off, globally.

Reply:
What's funny is that this might indeed be unused. I might try to refactor this part as a follow up, perhaps this is no longer needed.
not in this PR though, I'm leaving it this way for now.

Reply:
> or turning the functionality off, globally.

No option to only have it for private things etc. or exclude mill files? Was it working before actually, I see a lot of unused things removed by scalafix, so something feels off here.